tests/t288_arg_oct.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+from runtest import TestBase
+
+class TestCase(TestBase):
+    def __init__(self):
+        TestBase.__init__(self, 'arg_oct', result="""
```

Comment:
Nit, I prefer using `-` instead of `_` in the test file name (`arg-oct`).  The test case is different though, but please follow the convention.

Reply:
Done. :)